backend/src/models/User.js

```diff
@@ -49,16 +49,15 @@ const userSchema = new mongoose.Schema({
 // Logging the user in 
 userSchema.statics.findByCredentials = async (email, password) => {
     const user = await User.findOne({ email })
```

Comment:
what is this for. im not sure i understand what this does.

Reply:
this function is being used when the user is trying to login

Reply:
isnt that for the /login end point ? why is it in the models/user file ?

Reply:
we might use this function again in the future, for example if the user wants to change his password so we first ask him about the old one and consequently we will call this function to compare the stored one with what he wrote